.buildkite/pipeline.verify.yml

```diff
@@ -4,58 +4,70 @@ env:
   BUILDKITE_PLUGIN_VAULT_ENV_SECRET_PREFIX: "secret/data/buildkite/env"
 
 steps:
+  - group: ":white_check_mark: Validation"
+    # Validations are basic sanity checks, mostly around ensuring
+    # various configuration files are in proper working order.
+    key: validation
+    steps:
+    - label: ":jeans: All files are covered by Pants"
```

Comment:
There's actually [a real `:pants:` emoji](https://github.com/buildkite/emojis/pull/374) we can use now; I just haven't gotten around to doing the global search and replace 😅